<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recommend</class>
 <widget class="QWidget" name="recommend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Recommend</string>
  </property>
  <property name="styleSheet">
   <string notr="true">recommend
{
	background-color:#0a0612;
}
QPushButton
{
	background-color:#8a02b2;
	color:#1f1137;
	border-radius:20px;
}
QPushButton:hover
{
	background-color:#341d5c;
	color: #ffffff;
}
QLabel
{
	color:#ad03de;
}
QLineEdit
{
	background-color:#673ab7;
	color:#ffffff;
	border-radius:16px;
}
QLineEdit:hover
{
	background-color:#5d34a5;
}
QPlainTextEdit
{
	color:#ffffff;
	background-color:#3e236e;
	border-radius:12px;
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>781</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="pte3">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPlainTextEdit" name="pte6">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_follow3">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="pte2">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_follow1">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_follow6">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="pte1">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPlainTextEdit" name="pte5">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_follow4">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_follow5">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPlainTextEdit" name="pte4">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_follow2">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>570</y>
     <width>162</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
